services/service.ts

```diff
@@ -600,9 +612,20 @@ function runService() {
    * Executes a shell command and responds with exit code, stdout and stderr.
    */
   type ExecPayload = { command: string };
-  service.register('exec', (message) => {
+  service.register('exec', (message: Message) => {
+    if (!('command' in message.payload)) {
+      message.respond(makeError('missing "command"'));
+      return;
+    } else if (typeof message.payload['command'] !== 'string') {
```

Comment:
CI suggestion to omit `else` here sounds sensible.

Reply:
Where are these even coming from? I ported everything from the old `.eslintrc.json`, and I'm not getting these locally.

Reply:
Plus the default on [`no-else-return`](https://eslint.org/docs/latest/rules/no-else-return) is to allow `else if`.